src/main/kotlin/fi/metatavu/timebank/api/forecast/ForecastService.kt

```diff
@@ -63,6 +63,19 @@ class ForecastService {
         ).toList()
     }
 
+    /**
+     * Finds person by id from forecast
+     *
+     * @param personId Int
+     * @return ForecastPerson
+     */
+    fun findPerson(personId: Int): ForecastPerson? {
+        return jacksonObjectMapper().readValue(
+            doRequest("/v2/persons/$personId"),
+            ForecastPerson::class.java
```

Comment:
Also it would probably make sense to move the deserialization process across the methods of this class to `ForecastService::doRequest`.

```kotlin
private inline fun <reified T>doRequest(path: String): T {
    return try {
        ...
        when (response.code()) {
            200 -> jacksonObjectMapper().readValue(response.body()?.string(), T::class.java)
```

Reply:
Then for example `ForecastService::findPerson` can be simplified down to

```kotlin
fun findPerson(personId: Int): ForecastPerson? {
    return doRequest("/v2/persons/$personId")
}
```

and `ForecastService::getPersons`
```kotlin
fun getPersons(): List<ForecastPerson> {
    return doRequest("/v2/persons")
}
```

and obviously same thing for rest of the methods.